api/envoy/extensions/filters/http/alternate_protocols_cache/v3/alternate_protocols_cache.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.alternate_protocols_cache.v3;
+
+import "envoy/config/core/v3/protocol.proto";
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.alternate_protocols_cache.v3";
+option java_outer_classname = "AlternateProtocolsCacheProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Alternate Protocols Cache]
+
+// Configuration for the alternate protocols cache HTTP filter.
```

Comment:
From an API/end-user perspective, it's not clear to me what this filter does or why I would configure it. Should there be other documentation, release note, etc.? Or is this part of some larger body of work that will be documented later?

/wait-any

Reply:
Some of this was documented in PR#16385, including the description of the AlternateProtocolsCacheOptions messasge, but I'm happy to add any more that might make sense. In any case, this filter reads the alt-svc header and, via the alternate protocols cache, enables HTTP/3 to servers which advertise HTTP/3. Should I add more information here? Any suggestions welcome!

Reply:
This is essentially going to show up in the docs as a new filter with no documentation, so I think we should probably have something more here. Some thoughts:
1) Add something about this to the H3 documentation somewhere in arch overview?
2) Add a release note if it's something users could use now?
3) Perhaps add some new arch overview section on upstream H3 features?
4) Maybe some example config somewhere if it makes sense?

Do you mind circling around with @alyssawilk tomorrow and discussing some additional docs as she did the recent H3 doc pass? I don't know the state of all of this stuff, and you don't need to go crazy, but I think there should be at least either a) more complete documentation or b) some kind of WIP documentation saying that more docs will be coming.

/wait

Reply:
Will do! Thanks

Reply:
I second @mattklein123 suggestion to add a config example (example: https://github.com/envoyproxy/envoy/blob/main/docs/root/configuration/http/http_filters/ext_authz_filter.rst) and add a link to it from here, as this (along with other HTTP/3 configurations) is non-trivial to an end-user.
I guess this can be done in a followup PR, as it is still hidden.